en/asgardeo/docs/apis/restapis/actions.yaml

```diff
@@ -0,0 +1,745 @@
+openapi: 3.0.0
+info:
+  version: v1
+  title: Asgardeo - Action Management Rest API
+  description:
+    "This is the RESTful API for managing action configuration for an organization in WSO2 Asgardeo. \n
+    This API allows creating, retrieving, updating and deleting the action configurations. \n\n
+    To access Action Management APIs in Asgardeo, you need to first [get an access token](https://wso2.com/asgardeo/docs/apis/authentication/#get-an-access-token) from your organization."
+servers:
+  - url: 'https://api.asgardeo.io/t/{organization-name}/api/server/v1'
+security:
+  - OAuth2: []
+paths:
+  /actions/types:
+    get:
+      tags:
+      - Actions Endpoint
+      summary: List Action Types
+      description: "This API returns the detailed summary of the action types.\n\n
+      - <b>Scope(Permission) required:</b> ``internal_action_mgt_view``\n\n"
+      operationId: getActionTypes
+      responses:
+        '200':
+          description: Action Types are found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ActionTypesResponse'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
```

Comment:
example curls are missing for each endpoint

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4698/commits/69f8f822c61bc848415056485d62bc91980b7bc0